regression/goto-instrument/chain.sh

```diff
@@ -38,6 +38,7 @@ elif echo $args | grep -q -- "--dump-c-type-header" ; then
   cat "${name}-mod.gb"
   mv "${name}.gb" "${name}-mod.gb"
 elif echo $args | grep -q -- "--dump-c" ; then
+  cat "${name}-mod.gb"
```

Comment:
Is this going to cause issues with other tests / increase the amount of output drastically?

Reply:
Since this output is only generated when `--dump-c` is included in the test options, I think only very few cases (with small source files) will be affected. Thus, we should be fine.